htdocs/core/lib/security.lib.php

```diff
@@ -382,7 +382,7 @@ function checkUserAccessToObject($user, $featuresarray, $objectid=0, $tableandsh
 		$checkother = array('contact','agenda');	 // Test on entity and link to third party. Allowed if link is empty (Ex: contacts...).
 		$checkproject = array('projet','project'); // Test for project object
 		$checktask = array('projet_task');
-		$nocheck = array('barcode','stock');	// No test
+		$nocheck = array('barcode','stock','expensereport');	// No test
```

Comment:
Why not adding 'expensereport' into $check array instead ?

Reply:
@eldy you're right, i just did it and it corrects my problem